src/main/java/lab3/game/Board.java

```diff
@@ -1,5 +1,162 @@
 package lab3.game;
 
+import java.util.*;
+
+import static lab3.game.winRecord.callScore;
+import static lab3.ui.swingConsole.currentPlayer;
+
 public class Board {
-    // TODO: encapsulate the representation of the tictactoe board and provide instance methods to access and update it
+    private static final Map<Coordinates, String> tileMap = new HashMap<>();
+    private static final List<Coordinates> availableCoords = new ArrayList<>();
```

Comment:
These should really be instance variables.  You want to be able to create `Board` objects, and then use the instance variables to track the state of the board.  What you have here become global variables that you need to refer to throughout your code, making your code more difficult to understand.